.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Thank you for opening an issue. Please note that we try to keep the issue
+tracker reserved for bug reports and feature requests. For general usage
+questions, please see:
+https://github.com/digitalocean/terraform-provider-digitalocean/discussions
+
+**NOTE: Before submitting a bug**
+
+There are cases where the provider receives HTTP Service Error (500 level HTTP
+statuses) responses from the DigitalOcean API. There are some cases where the
+provider might handle these and retry. If the problem persists, it's best to
+contact [DigitalOcean support](https://cloudsupport.digitalocean.com/) 
+
+# Bug Report
+
+Include as much of the following details with your bug report:
+
+> everything above this can be ommitted
```

Comment:
One thing I've done in the past is wrap stuff like the above in HTML comment tags. That way it doesn't actually show if the user doesn't remove it.

E.g. https://github.com/digitalocean/doctl/blame/main/.github/ISSUE_TEMPLATE/bug_report.md#L12

Reply:
Good idea